09_simulations/models.py

```diff
@@ -162,12 +173,105 @@ def train_mlp(X_train,
     return cv_pipeline
 
 
+def train_linear_csd(X_train,
+                     y_train,
+                     train_domains,
+                     seed,
+                     n_domains=None,
+                     params=None,
+                     latent_dim=2,
```

Comment:
There's also a latent_dim parameter within `params`, will this cause problems?

Reply:
Removed! You're right, the function argument wasn't being used.